RZBRP100/iocBoot/iocRZBRP100-IOC-01/st-common.cmd

```diff
@@ -0,0 +1,44 @@
+epicsEnvSet "STREAM_PROTOCOL_PATH" "$(RZBRP100)/data"
+epicsEnvSet "DEVICE" "L0"
+
+##ISIS## Run IOC initialisation 
+< $(IOCSTARTUP)/init.cmd
+
+## Device simulation mode IP configuration
+$(IFDEVSIM) drvAsynIPPortConfigure("$(DEVICE)", "localhost:$(EMULATOR_PORT=57677)")
+
+## For recsim:
+$(IFRECSIM) drvAsynSerialPortConfigure("$(DEVICE)", "$(PORT=NUL)", 0, 1, 0, 0)
+
+## For real device:
+$(IFNOTDEVSIM) $(IFNOTRECSIM) drvAsynSerialPortConfigure("$(DEVICE)", "$(PORT=NO_PORT_MACRO)", 0, 0, 0, 0)
```

Comment:
Wiki says this is a USB device but this looks like standard serial port setup - which is it?

Reply:
Both.  The USB driver creates a virtual COM port on the host, enabling it to be accessed as a standard serial port.  More details have been added to the [Wiki page](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Razorbill-RP100-Strain-Cell-PSU).